warehouse/templates/manage/release.html

```diff
@@ -126,7 +126,9 @@ <h3>Delete release</h3>
         files
       {% endtrans %}
     {% else %}
-      Deleting will irreversibly delete this release
+      Deleting will irreversibly delete this release.
+      You will not be able to re-upload a new distribution of the same type with the same version number.
+      Consider making a new release or a <a href="https://www.python.org/dev/peps/pep-0440/#post-releases">post release</a> instead.
```

Comment:
These two lines need to be moved below the `{% endif %}` below.

Reply:
ok, fix it now